SwiftTweaks/TweakButton.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  TweakButton.swift
+//  SwiftTweaks
+//
+//  Created by Jarosław Pendowski on 28/04/2017.
+//  Copyright © 2017 Khan Academy. All rights reserved.
+//
+
+import UIKit
+
+class TweakButton: UIButton {
+	
+	typealias ControlState = UInt
+	
+	private var borderColors: [ControlState: UIColor] = [:]
+	override var isHighlighted: Bool {
+		didSet { updateState() }
+	}
+	
+	override var isEnabled: Bool {
+		didSet { updateState() }
+	}
+	
+	var borderWidth: CGFloat = 2 {
+		didSet { updateState() }
+	}
+	
+	var cornerRadius: CGFloat = 4 {
+		didSet { updateState() }
+	}
+	
+	func setBorderColor(_ color: UIColor, for state: UIControlState = .normal) {
+		borderColors[state.rawValue] = color
+	}
+	
+	override func didMoveToWindow() {
```

Comment:
So at that time I was a bit lazy and decided that it's for internal use only, so writing `init`s, especially the `coder` one is a pain for an internally used button.